nikola/post.py

```diff
@@ -161,6 +163,14 @@ def __init__(
             for lang in sorted(self.translated_to):
                 default_metadata.update(self.meta[lang])
 
+        # Find paths from metadata
+        self.folders = {lang: self.meta[lang].get('path', self.folder_relative) for lang in self.config['TRANSLATIONS'].keys()}
+        if isinstance(self.folder_base, (utils.bytes_str, utils.unicode_str)):
+            self.folders = {lang: os.path.normpath(os.path.join(self.folder_base, folder)) for lang, folder in self.folders.items()}
+        elif self.folder_base is not None:
+            self.folders = {lang: os.path.normpath(os.path.join(self.folder_base(lang), folder)) for lang, folder in self.folders.items()}
+        self.folder = self.folders[self.default_lang]
+
```

Comment:
Yes, that's the current behavior.

Whether the name makes sense: I don't know. This is just a test implementation for the idea in #2116.